explainer.md

```diff
@@ -120,7 +121,7 @@ partial interface XRSession {
 
 `hitTest` return value
 *   The hit results are returned in sorted order with the nearest intersection to the origin of the ray at the beginning of the array and the furthest intersection from the origin of the ray at the end of the array.
-*   XRHitResult.hitMatrix is a 4x4 matrix where the translation represents the position where the ray hit the object and the orientation has a Y-axis that corresponds with the normal of the object at that location.
+*   XRHitResult.hitMatrix is a 4x4 matrix where the translation represents the position where the ray hit the object. The orientation of the matrix has axes where Y+ points along the estimated surface normal, X+ is perpendicular to the cast ray and parallel to the physical surface centered around the hit test, and Z+ points roughly toward the user's device (see image at top of explainer). If the hit-test ray and the surface normal are almost parallel, very small input variations can result in large changes in the result orientation. In the case that they are exactly parallel, the rotation of the X and Z axes is non-deterministic. This means that the normal use-case of placing objects on a horizontal surface will result in the object facing the user. In the case where you are pointing directly at a wall, the orientation may fluctuate wildly and should probably be ignored by the app.
```

Comment:
Wouldn't the orientation fluctuate the same way if the device is pointing to the ground perpendicularly?

Reply:
That is true - the ARCore team determined that pointing directly at the floor was much less likely than pointing directly at a wall in front of you. I can edit the explainer text to mention this.